src/qt/forms/debugwindow.ui

```diff
@@ -678,14 +850,8 @@
            </item>
            <item>
             <widget class="QLabel" name="lblGraphRange">
-             <property name="minimumSize">
-              <size>
-               <width>100</width>
-               <height>0</height>
-              </size>
-             </property>
-             <property name="alignment">
-              <set>Qt::AlignCenter</set>
+             <property name="text">
+              <string>Range</string>
```

Comment:
Why this change?

This label will be overwritten in https://github.com/bitcoin-core/gui/blob/480bf01c295527bd212964efe4df3bb886db5654/src/qt/rpcconsole.cpp#L1006

and it is unnecessary burden to translate the "Range" string.